src/components/ButtonComponents/NumberButtons/NumberButton.js

```diff
@@ -1,9 +1,14 @@
 import React from "react";
 
-const NumberButton = () => {
+const NumberButton = props => {
```

Comment:
This is a valid way to pass in props, though destructuring them (e.g. instead of props, you could write {num}) is a good habit to form because it enhance readability.